README.md

```diff
@@ -17,30 +17,45 @@ Features
   component classes for production builds
 
 More information on why that is useful are available on our
-[blog](http://mainmatter.com/blog/2016/03/04/ember-test-selectors.html)!
+[blog](https://mainmatter.com/blog/2017/11/17/ember-test-selectors-road-to-1-0/)!
 
 
 Compatibility
 ------------------------------------------------------------------------------
 
-- Ember 3.8 or above
-- Ember CLI 3.8 or above
-- Node.js 12 or above
+- Classic Ember apps: Ember 3.8 or above, Ember CLI 3.8 or above
+- Embroider+Vite apps: Ember 3.28 or above, Ember CLI 4.12 or above
+- Node.js 18 or above
 
 
-Installation
+Installation in a Classic Ember app
 ------------------------------------------------------------------------------
 
+Install and use `ember-test-selectors` addon:
+
 ```bash
 ember install ember-test-selectors
 ```
 
+See the [addon configuration in the dedicated README](https://github.com/mainmatter/ember-test-selectors/blob/master/ember-test-selectors/README.md).
+
+
+Installation in an Embroider+Vite app
+------------------------------------------------------------------------------
+
+It's recommended to install and configure `strip-test-selectors` Babel plugin directly. (You can install and use `ember-test-selectors` classic addon, but `@embroider/compat` will have to rewrite it to maintain the compatibility with Vite, and this step has a performance cost.)
```

Comment:
```suggestion
It's recommended to install and configure `strip-test-selectors` AST Transform directly. (You can install and use `ember-test-selectors` classic addon, but `@embroider/compat` will have to rewrite it to maintain the compatibility with Vite, and this step has a performance cost.)
```

Reply:
yes we configure this in the babel config but it's actually an AST transform plugin for `babel-plugin-ember-template-compilation`

Reply:
I wrote this comment before I got to the whole README of the plugin, I didn't realise we had JS plugins too 🙈 🫠

Reply:
I see a place where I can include this precision though 👍